starcheck/src/starcheck.pl

```diff
@@ -1338,8 +1332,9 @@ =head1 DESCRIPTION
 results in a fatal error, however.
 
 Starcheck uses the SKA environment variable to locate the default agasc file
-"${SKA}/data/agasc/proseco_agasc_1p7.h5".  If SKA is not set this defaults to
-'/proj/sot/ska'.
+"${SKA}/data/agasc/proseco_agasc_*.h5".  If SKA is not set this defaults to
+'/proj/sot/ska'.  The agasc file location can be controlled with the agasc module
```

Comment:
I suspect that the new code will raise an exception if `/proj/sot/ska` bit is not set. The starcheck perl code does `my $SKA = $ENV{SKA} || '/proj/sot/ska';` but that won't help the `agasc` module which requires that the `SKA` env var be set. So I would just update to remove the `If SKA is not set ...`.

Reply:
OK, I updated the docs here.  Since the Python side doesn't work without SKA set, it might make sense to not use the default on the perl side or pass the default over to the Python side.  But for now, just cutting this in the docs seems fine.